src/DGtal/topology/CubicalComplexFunctions.h

```diff
@@ -431,13 +431,27 @@ namespace DGtal
     template <typename TKSpace, typename TCellContainer,
               typename BdryCellOutputIterator,
               typename InnerCellOutputIterator>
-    void 
-    filterCellsWithinBounds( const CubicalComplex< TKSpace, TCellContainer > & K, 
-                             const typename TKSpace::Point& kLow,  
+    void
+    filterCellsWithinBounds( const CubicalComplex< TKSpace, TCellContainer > & K,
+                             const typename TKSpace::Point& kLow,
                              const typename TKSpace::Point& kUp,
-                             BdryCellOutputIterator itBdry, 
+                             BdryCellOutputIterator itBdry,
                              InnerCellOutputIterator itInner );
 
+    /**
+     * Create object from the spels in the complex.
+     *
```

Comment:
Mmm.. I'm not sure to understand the function from the description..  If you are returning a DigitalObject (set + topo) it should be stated a bit more clearly (as well as the convention that spel=n-cel <-> grid points in Z^n) maybe

Let me check the code

Reply:
Updated to clarify spel and that the user has to provide a compatible TObject type for the method.